Source/SwiftLintFramework/Rules/VariableNameRule.swift

```diff
@@ -78,11 +79,14 @@ public struct VariableNameRule: ASTRule {
     public let example = RuleExample(
         ruleName: "Variable Name Rule",
         ruleDescription: "Variable name should only contain alphanumeric characters, " +
-        "start with a a lowercase character and be between 3 and 40 characters in length.",
+        "start with a a lowercase character and be between 3 and 40 characters in length. " +
+        "In an exception to the above, variable names may start with a capital letter when " +
+        "they are declared static and immutable.",
         nonTriggeringExamples: [
             "let myLet = 0",
             "var myVar = 0",
-            "private let _myLet = 0"
+            "private let _myLet = 0",
+            "static let MyLet = 0"
```

Comment:
This is the reason the tests are failing: it looks like when a `static let` is at a global level, SourceKitten doesn't actually consider it static? (Which is pretty smart, actually.)

I can fix this here by making it `"class Abc { static let MyLet = 0 }"`, but then it's inconsistent with the other examples. Let me know what your preferred way of dancing around this kind of thing is.